pages/linux/turbostat.md

```diff
@@ -0,0 +1,24 @@
+# turbostat
+
+> Report processor frequency and idle statistics.
+> More information: <https://manned.org/turbostat.8>.
+
+- Display help:
```

Comment:
The style guide highly recommends us to put help and version commands, in this order, as the last examples for not polluting the first examples.

Remember that the goal of tldr pages is to provide the most useful example usages of a command (or subcommand).